data/net/minecraft/world/entity/ai/goal/Goal.mapping

```diff
@@ -1,13 +1,13 @@
 CLASS net/minecraft/world/entity/ai/goal/Goal
 	METHOD canContinueToUse ()Z
-		COMMENT Returns whether an in-progress EntityAIBase should continue executing
+		COMMENT Returns whether the goal should continue executing
```

Comment:
Should use `@return`

Reply:
Wasn't sure to what extent these sort of @ annotations for javadocs are allowed

Reply:
Per [the standards](https://parchmentmc.org/docs/mappings.html):
- Don't use `@since`, `@author`, or `@param` (the last one is handled via parameter-level comments
- If you use `@link` or `@see` they need to be using fully qualified class names
- All others can be used if there's a good reason to.